Assets/Plugins/Source/Editor/Utility/GUIEditorUtility.cs

```diff
@@ -96,94 +96,6 @@ public static void AssigningTextField(string label, ref string value, float labe
             EditorGUIUtility.labelWidth = originalLabelWidth;
         }
 
-        public static void AssigningPath(string label, ref string filePath, string prompt, string directory = "", string extension = "", bool selectFolder = false, bool horizontalLayout = true, float maxButtonWidth = 100, float labelWidth = -1, string tooltip = null)
-        {
-            if (horizontalLayout)
-            {
-                EditorGUILayout.BeginHorizontal();
-            }
-
-            AssigningTextField(label, ref filePath, labelWidth, tooltip);
-
-            bool buttonPressed = maxButtonWidth > 0 ? GUILayout.Button("Select", GUILayout.MaxWidth(maxButtonWidth)) : GUILayout.Button("Select");
-
-            if (buttonPressed)
-            {
-                var newFilePath = selectFolder ? EditorUtility.OpenFolderPanel(prompt, "", "") : EditorUtility.OpenFilePanel(prompt, directory, extension);
-                if (!string.IsNullOrWhiteSpace(newFilePath))
-                {
-                    filePath = newFilePath;
-                }
-            }
-
-            if (horizontalLayout)
-            {
-                EditorGUILayout.EndHorizontal();
-            }
-        }
-
-        private delegate T InputRenderDelegate<T>(string label, T value, float labelWidth, string tooltip);
```

Comment:
This was relocated to another place within the class definition that makes more sense.